src/__experimental__/components/date/date.stories.js

```diff
@@ -167,4 +167,4 @@ storiesOf('Experimental/Date Input', module)
   .add(...makeStory('classic', classicThemeSelector, dateComponent, true))
   .add(...makeStory('empty', dlsThemeSelector, EmptyDateComponent))
   .add(...makeStory('validations', dlsThemeSelector, ValidationDateComponent))
-  .add(...makeStory('autoFocus', dlsThemeSelector, autoFocusDateComponent));
+  .add(...makeStory('autoFocus', dlsThemeSelector, autoFocusDateComponent, true));
```

Comment:
Why we excluded `autoFocus` stories ?

Reply:
I want to ask, why **only** autofocus is excluded, not all of them ?

Reply:
It's only the auto focus that is rendering differently each time. If it changes on each PR it's not giving us any confidence.